tests/vmi_iothreads_test.go

```diff
@@ -133,9 +133,7 @@ var _ = Describe("IOThreads", func() {
 		})
 
 		table.DescribeTable("[ref_id:2065] should honor auto ioThreadPolicy", func(numCpus int, expectedIOThreads int) {
-			if numCpus > availableCPUs {
-				Skip(fmt.Sprintf("Testing environment does not contain a node with required %d CPUs number, the highest detected number is %d", numCpus, availableCPUs))
-			}
+			Expect(numCpus <= availableCPUs).To(BeTrue(), "Testing environment should have at least one node with required CPUs number")
```

Comment:
I would recommend to put the number of required CPUs into the message, i.e. reuse the `fmt.Sprintf` that you used before. This way the recipient exactly knows what the requirements are. I.e. sth like:

    Expect(numCpus <= availableCPUs).To(BeTrue(), 
        fmt.Sprintf("Testing environment only has nodes with %d CPUs available, but required are %d CPUs", availableCPUs, numCpus))

Reply:
:+1:

Reply:
good point, done.